config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   get "/", to: "application#welcome"
 
+  get "/admin/shelters", to: "admin/shelters#index"
+
+  post "/pet_applications", to: "pet_applications#create"
```

Comment:
Excellent RESTful routing! Another RESTful route here can include the IDs like so if you don't want to deal with sending hidden values: `/applications/:app_id/pets/:pet_id`
Either path is fine and RESTful!